.github/ISSUE_TEMPLATE/out_of_date_package_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Out-of-date package reports
+about: For packages that are out-of-date
+title: ''
+labels: '0.kind: out-of-date package'
+assignees: ''
+
+---
+
+
+###### Checklist
+
+<!-- Note that these are hard requirements -->
+
+<!--
+You can use the "Go to file" functionality on github to find the package
+Then you can go to the history for this package
+Find the latest "package_name: old_version -> new_version" commit
+The "new_version" is the the current version of the package
```

Comment:
While this is mostly correct, people should really look into the `.nix` file and look for `version= `.
But the first check should actually be on https://search.nixos.org/packages?channel=unstable as the versions there should be up-to-date with nixpkgs (at most few days behind, when there is something seriously broken).

Reply:
Hi! I took as an example pharo.
It has no version on the search website and the version for the pharo launcher is a date and (I don't think, I'm going to check tommorow) not an official version number like 5, 6 etc like on the actual commit.

Also, for a package like discord, having the wrong version means having no discord app for a few hours. If the PR of the new version of discord is added fast as it should be, an out-of-date issue will more than surely be raised if the person doesn't check the master branch even if hydra takes two or less hours.

I thought taking the path where there is less risk of raising issues might be better, however I understand the fact that it's way easier to check the version number for the package on the site :/

It would be good if we could search the master branch on the site, it would be really usefull. If this were the case, I would a 100% use this method.

Such a feature could also actually be pretty good outside of this issue. Should I make another issue on this subject?

Reply:
I just checked and yes, the current version of the pharo-launcher is 2.2.
It's not written in a date format unfortunately